proposals/LP0004-project-governance.md

```diff
@@ -0,0 +1,426 @@
+
+## Voting Process
+
+Public notice of any vote will be published on the LLVM discussion forums under the "Elections" topic at least 14 days before voting opens with a schedule for the election. Each election will be open for 14 days. Reminders will be posted on the "Elections" topic at:
+
+* 7 days before voting opens.
+* The day voting opens.
+* 7 days after voting opens.
+* 48 hours before voting closes.
+* 24 hours before voting closes.
+
+Voter eligibility will be determined 7 days before voting opens by the rules for defining _voting contributors_. Voting will be executed using a private poll hosted by [Concordcet Internet Voting Service]([https://civs1.civs.us/](https://civs1.civs.us/)). Concordcet uses a ranked choice polling process.
+
+> Note: Concordcet Internet Voting Service suggested here is a placeholder. A more through review of voting systems needs to be conducted. This will overlap with voter registration data collection.
+
+All eligible _voting contributors_ will be contacted via email with instructions on how to vote on the day voting opens.
+
+### Elections
+
+The LLVM Foundation will act as a facilitator for community elections. The LLVM Foundation will maintain a voter registration database in compliance with local and international privacy laws. The LLVM Foundation will publish a privacy policy relating to the voter registration system.
+
+Elections for _area teams_ occur in January of each year. A two week long nomination period begins the second Monday in January. During the nomination period any _voting contributor_ can nominate themselves or another _voting contributor_ to run for any one _area team_. No individual can run for more than one _area team_ in a single election. An individual nominated for more than one area team will be responsible for choosing which team they want to run for. Nominations will be recorded publicly for community visibility. Unsuccessful results in an election do not impact nomination eligibility in subsequent elections.
+
+Voting begins the fourth Monday in January and continues for 2 weeks. Election results will be announced no later than the second Monday in February. The term of the newly elected _area team_ begins the first Monday in March. Each _area team_ will meet during the first week in March to elect from themselves the team secretary and chair to re-constitute the _project council_.
+
+## Amending This Document
+
+This document should be amended through public proposals following the [LP0001 LLVM Decision Making](https://github.com/llvm/llvm-www/blob/main/proposals/LP0001-LLVMDecisionMaking.md) process. The _project council_ shall serve as the review managers for any proposal to amend this document.
+
+## Impact On Other Projects
+
+This proposal impacts all LLVM projects and all contributors to the LLVM community.
+
+## Frequently Asked Questions
+
+### Why create new structures instead of expanding the role of code owners?
```

Comment:
llvm/clang/lldb have switched from "code owners" to "maintainers". This might need rewording.

Reply:
Yes, this has been discussed and @llvm-beanz said he would update to the new wording.